src/meteofrance/helpers.py

```diff
@@ -27,7 +31,9 @@ def get_warning_text_status_from_indice_color(int_color: int, lang: str = "fr")
         return ALERT_COLOR_LIST_EN[int_color]
 
 
-def get_phenomenon_name_from_indice(int_phenomenon: int, lang: str = "fr") -> str:
+def get_phenomenon_name_from_indice(
+    int_phenomenon: int, lang: str = "fr"
+) -> Optional[str]:
```

Comment:
same here for else and optional

Reply:
else fixed.
Optional same as previous discussion.